packages/dialtone-vue2/recipes/item_layout/contact_info/contact_info.vue

```diff
@@ -28,13 +28,13 @@
             image-alt=""
             :overlay-text="avatar.text"
             :avatar-class="[{ 'd-mln24': index > 0, 'd-bc-brand': !!avatar.halo }]"
+            :show-icon="showAvatarIcon"
           >
             <template
               #icon="{ iconSize }"
             >
               <!-- @slot Slot for avatar icon in a list -->
               <slot
-                v-if="hasAvatarIcon"
```

Comment:
If I'm not mistaken, all we have to do is remove this condition and everything works correctly right?

Reply:
No,  `isIconType` in the avatar component would still not be reactive

Reply:
What we can do is change isIconType from a computed to a function

```
v-else-if="isIconType()"

...

methods: {
    isIconType () {
      return this.$scopedSlots.icon && this.$scopedSlots.icon();
    },
```

I tested this to make sure it works and it seems to.

before:
![2024-10-17 12 15 03](https://github.com/user-attachments/assets/eb143514-2241-4377-b410-5bec5b6e992e)

after:
![2024-10-17 12 16 29](https://github.com/user-attachments/assets/a5a9896d-7b75-4551-9793-a1011a2735a5)

Reply:
great, will try this tomorrow if I feel better, thank you!